src/main/java/duke/DukeStorage.java

```diff
@@ -0,0 +1,110 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+import duke.exception.DukeArgumentException;
+import duke.exception.DukeFileException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.TaskList;
+import duke.task.ToDo;
+
+public class DukeStorage {
+    private static String tasksFile = "data/tasks.txt";
+    private static String dataFolder = "data/";
+    private static String partialLoadMessage = "you may have a corrupted/edited save file. Tasks partially loaded";
+    private static String successLoadMessage = "Tasks successfully loaded!";
+
```

Comment:
Same here, all these constants can be declared as final, you might want to follow the naming convention for constants (e.g. `TASKS_FILE`)

Reply:
Good points